.github/actions/test/action.yaml

```diff
@@ -15,3 +15,7 @@ runs:
     run: |
       poetry run pytest
     shell: bash
+  - name: Rerun workaround tests to check for incompatibilities
+    run: |
+      poetry run pytest tests/workarounds
+    shell: bash
```

Comment:
We need to run the test suite multiple times to ensure workarounds are functioning